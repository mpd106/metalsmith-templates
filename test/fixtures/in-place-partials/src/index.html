---
name: First Last
---

{{> nav}}
{{name}}
