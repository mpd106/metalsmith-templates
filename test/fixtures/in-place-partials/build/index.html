
templates/partials/nav.htmlFirst Last
