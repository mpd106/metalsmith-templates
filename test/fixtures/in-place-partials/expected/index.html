
<p>Nav</p>
First Last
